swagger/error-with-secrets.json

```diff
@@ -0,0 +1,110 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Error with Secrets",
+    "description": "Tests whether loggers/tracers redact secrets and PII within error responses.",
+    "version": "1.0.0"
+  },
+  "host": "localhost:3000",
+  "schemes": ["http"],
+  "produces": ["application/json"],
+  "paths": {
+    "/errorWithSecrets": {
+      "get": {
+        "operationId": "GetErrorWithSecrets",
+        "description": "Gets an error response containing secrets and PII.",
+        "responses": {
+          "default": {
```

Comment:
This swagger means all calls are failure, autorest refuses this and won't generate in some language since that's an impossible scenario. At least add a 204 with no body (that's weird for a GET, but least describe a success scenario)